### DISCUSSION TITLE
saf-training/courses/guidance/03

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#271] [Ideas]
# saf-training/courses/guidance/03

2.1 Security Guidance Before we go further, let's discuss what we mean by 

https://mitre.github.io/saf-training/courses/guidance/03.html

**Comment by karikarshivani** [comment]
Maybe the chart in 3.3 can be replaced with a mermaid chart to standardize the content (also make it more readable)

**Comment by mebric** [comment]
FYI there is a mis spelling in 3.1, line 2 "creating".  TY

**Comment by p-oneil** [comment]
Moved to PR
